bigframes/session/__init__.py

```diff
@@ -1365,6 +1373,12 @@ def read_gbq_function(
                 Whether the function is a row processor. This is set to True
                 for a function which receives an entire row of a DataFrame as
                 a pandas Series.
+            output_type (type, default None):
+                Python type equivalent to the BigQuery return type. This is used
+                to coerce the return value of the BigQuery function to BigFrames
+                type. For example, if the BigQuery function returns a JSON
+                serialized array of integers such as "[1, 2, 3]", then user can
+                set `output_type=list[int]` to read it as array of integers [1, 2, 3].
```

Comment:
I don't know if this makes sense for existing bq functions. Its more helpful for local python functions where the output type is otherwise ambiguous. Users should call astype or similar methods to explicitly coerce the result.

Reply:
The place where it would be useful is reading back an remote function that was created using bigframes.  When they used it then their code didn't have to call astype, so we want to give them a way to have that code continue to work after reading back. The test `test_df_apply_axis_1_array_output` captures this usecase.

Reply:
Remove this param in the latest change, PTAL